HW1/P3/P3.py

```diff
@@ -0,0 +1,13 @@
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext()
+
+wlist = sc.textFile('./EOWL_words.txt')
+
+wlist2 = wlist.map(lambda line: (''.join(sorted(line)), (line)))
+
+wlist3 = wlist2.groupByKey().mapValues(lambda x: [i for i in x ]).map(lambda y: (y[0],len(y[1]),y[1]))
```

Comment:
As a good rule of practice, you probably want to avoid groupByKey and use ReduceByKey instead